mutmut/cache.py

```diff
@@ -1,13 +1,13 @@
-# coding=utf-8
+#!/usr/bin/env python
```

Comment:
Do we really need shebang line in `cache.py`? I don't see `cache.py` being used as a standalone script. This stands for all shebangs in this PR, before adding them, ask yourself "is this file is ever going to be ran as a standalone".